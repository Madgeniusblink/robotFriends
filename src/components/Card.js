import React from 'react';


const Card = ({name, email, id, set}) => {
    return (
        <div className="tc bg-purple dib br3 pa3 ma2 grow bw2 shadow-5">
            <img alt='robots' src={`https://robohash.org/${id}?${set}&100x100`} ></img>
            <div>
                <h2>{name}</h2>
                <p>{email}</p>
            </div>
        </div>
    )
}

export default Card;
