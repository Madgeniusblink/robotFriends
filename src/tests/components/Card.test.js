import { shallow, mount, render } from 'enzyme';
import React from 'react'
import Card from '../../components/Card'
import '../setupTest'



it('expect card component length to be one', () => {
    const wrapper = shallow(<Card />);
    expect(wrapper.length).toMatchSnapshot()
})

