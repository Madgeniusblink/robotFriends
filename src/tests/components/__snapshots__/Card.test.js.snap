// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect card component length to be one 1`] = `1`;
